src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,207 @@
+.header {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+
+  &__logo {
+    padding: 0 50px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+}
+
+.nav__item {
+  list-style: none;
+  position: relative;
+}
+
+.nav__item:last-child {
+  padding-right: 50px;
+}
+
+.logo-link {
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item:not(:last-child) {
+  margin: 0 20px 0 0;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding: 22.5px 0;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  line-height: 15px;
+  color: #060b35;
+
+  &:hover {
+    color: #00acdc;
+    transition: 0.3s;
+  }
+}
+
+.is-active {
+  color: #00acdc;
+
+  &::before {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 4px;
+    background-color: #00acdc;
+    border-radius: 8px;
+  }
+}
+
+.catalog {
+  display: flex;
+  flex-direction: row;
+  padding: 50px 40px;
+  flex-wrap: wrap;
+}
+
+.main__catalog {
+  margin: 0 auto;
+  max-width: 1000px;
+
+  &:first-child {
+    padding-left: 0;
+  }
```

Comment:
This block also should be in separate file.

`11.[FILE STRUCTURE] - You should create separate file for each BEM block`

detailed info how to do it in [this video](https://mate.academy/learn/css-advanced/bem#/video/917)